shed/event_streams.py

```diff
@@ -254,33 +257,31 @@ def curate_streams(self, nds, outbound):
         If we get multiple streams make (name, (doc, doc, doc, ...))
         Otherwise (name, (doc,))
         """
+        # if there are multiple streams
+        if isinstance(nds[0], tuple):
```

Comment:
Broke this out because the only difference between inbound and outbound curation is if we can give back `(name, doc)` (which is valid outbound but not inbound)